test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -43,4 +54,12 @@ private void assertAreUnique(Object... objects) {
     private void assertNotUnique(Object... objects) {
         assertFalse(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
+
+    private void assertExistNull(Object... objects) {
+        assertTrue(Utils.isAnyNull(objects));
+    }
+
+    private void assertNoNull(Object... objects) {
+        assertFalse(Utils.isAnyNull(objects));
+    }
```

Comment:
It's good to follow the style used by the code nearby 👍
Having said that, it is doubtful if extracting these two methods add any value. 
In assertAreUnique and assertNotUnique refactoring helps to convert arrays passed to the method as collections. 
For isAnyNull refactoring just adds on to the height of the code.